### PR TITLE
[IOS-540] 3D touch support for the older version of iPhones improvements

### PR DESCRIPTION
**Pull request by Malibris**
### Ticket
[IOS-540](https://netguru.atlassian.net/browse/IOS-540)

### Task Description
3D touch support for the older version of iPhones

### Aditional Notes 
Fixes: expanded area of sourceRect For PeekPop's `PreviewingContext` & added compatibility for all iPhones below 6s version

### CODE REVIEW

File: Inbbbox/Source Files/ViewControllers/ProfileViewController.swift [hunk 1]
```diff
@@ -449,14 +449,17 @@ extension ProfileViewController: PeekPopPreviewingDelegate {
...
-        previewingContext.sourceRect = cell.frame
         if let viewModel = viewModel as? UserDetailsViewModel {
             let controller = ShotDetailsViewController(shot: viewModel.shotWithSwappedUser(viewModel.userShots[indexPath.item]))
             controller.customizeFor3DTouch(true)
             controller.shotIndex = indexPath.item
-
+            previewingContext.sourceRect = cell.frame
             return controller
         } else if let viewModel = viewModel as? TeamDetailsViewModel, collectionView.collectionViewLayout is TwoColumnsCollectionViewFlowLayout {
+            let previewingContextMargin = CGFloat(4)
```
Comment (Siemian) [inline Inbbbox/Source Files/ViewControllers/ProfileViewController.swift]: the same code in three different places? Please move it to for example uiviewcontroller extension

Comment (Siemian) [inline Inbbbox/Source Files/ViewControllers/ProfileViewController.swift]: then just call right function here

Comment (Malibris) [inline Inbbbox/Source Files/ViewControllers/ProfileViewController.swift]: I think of replacing the PeekPop's 3D Touch emulation with entirely custom code in the next week, but I guess you are right - even temporary code should keep up with standards.

File: Inbbbox/Source Files/Extensions/UIViewExtension.swift [hunk 2]
```diff
@@ -14,4 +14,11 @@ extension UIView {
         viewForReturn.backgroundColor = color
         return viewForReturn
     }
+
+    static func extendedFrame(forFrame frame: CGRect) -> CGRect {
```
Comment (Siemian) [inline Inbbbox/Source Files/Extensions/UIViewExtension.swift]: maybe you could add comment what it does? without looking into implementation i would expect it to extend in all directions. Also describing why 4 is hardcoded would be nice

Comment (Malibris) [inline Inbbbox/Source Files/Extensions/UIViewExtension.swift]: Sure thing

**Pull request merged**